zeq/hbp/vocabulary.h

```diff
@@ -23,9 +37,46 @@ static const uint128_t EVENT_SELECTED_IDS(
     lunchbox::make_uint128( "hbp::SelectedIDsEvent" ));
 static const uint128_t EVENT_TOGGLE_ID_REQUEST(
     lunchbox::make_uint128( "hbp::ToggleIDRequest" ));
+static const lunchbox::uint128_t EVENT_REQUEST(
+    lunchbox::make_uint128( "hbp::RequestEvent" ));
+static const lunchbox::uint128_t EVENT_IMAGE_RAW_RGBA8(
+    lunchbox::make_uint128( "hbp::imageRawRGBA8Event" ));
 //@}
 
 /**
+ * Serialize the given rgba raw image into an Event of type EVENT_IMAGE_RAW.
+ * The image format must be raw rgba with 8bits per component.
+ * @param image the image.
+ * @return the serialized event.
+ */
+::zeq::Event serializeimageRawRGBA8( const ImageRawRGBA8& image );
```

Comment:
I would vote for the first one then to keep the API of this function stable if new/other attributes for the image are needed + the number of parameters are reduced + the symmetry with deserialize is kept.

But good catch, the function name with the uppercase I needs to be done